src/components/requirements/ChainIndicator.tsx

```diff
@@ -0,0 +1,16 @@
+import { Badge } from "@/components/ui/Badge";
+import { CHAINS } from "@/config/chains";
+import type { Register } from "wagmi";
+
+export const ChainIndicator = ({
+  chain,
+}: { chain: Register["config"]["chains"][number]["id"] }) => (
```

Comment:
What do you think, should we extract the `Register["config"]["chains"][number]["id"]` to a type somewhere?

Reply:
It looks like a common type, we can extract it into a wagmi type file.